swap_meet/item.py

```diff
@@ -1,2 +1,21 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self,condition =0, id = None, age = 0):
+        self.category = self.__class__.__name__
+        id = uuid.uuid4().int if id is None else id
```

Comment:
Something to consider, what happens here if someone tries to pass in an `id` that is not `None`, but is an `id` that is already assigned to something in inventory? What type of logic could we implement here in order to prevent that from happening?